### DISCUSSION TITLE
"Online" status should not count if not actively logging

### DISCUSSION THREAD

**Discussion by braininjurycoalition** [InternetConnectionMonitor/icm-public#123] [Bug reports]
The Log reports that from yesterday at 11:03:45 until today at 9:55:06 AM, I had 82275 seconds (22:51:15) straight of online connection. However, I was not online for the vast majority of the time - I was actually online and using Chrome for only about 2.5 hours. 

The log should report the percent of availability (and percent offline) **_out of the total time actively logging/monitoring_**, not the total number of time available out of the past day (24 hours). Thanks!

**Comment by dmishh** [comment]
Duplicate of #104
